src/background.js

```diff
@@ -2,12 +2,17 @@ let myPromises = [
     messenger.messageDisplayScripts.register({
         js: [
             { file: "/decoders.js" },
-
         ],
     }),
     messenger.messageDisplayScripts.register({
         js: [
             { file: "/change_message.js" }
         ],
+    }),
+    messenger.composeScripts.register({
+        js: [
+            { file: "/decoders.js" },
```

Comment:
You also need change_message.js here to adjust links in HTML messages.
```suggestion
            { file: "/decoders.js" },
            { file: "/change_message.js" },
```